projectile.el

```diff
@@ -672,11 +672,34 @@ means check all the subdirectories of DIRECTORY.  Etc."
   :type '(repeat (choice directory (cons directory (integer :tag "Depth"))))
   :package-version '(projectile . "1.0.0"))
 
+(defcustom projectile-fd-executable
+  (cond
+   ((executable-find "fdfind") "fdfind")
+   ((executable-find "fd") "fd"))
+  "Path or name of fd executable used by Projectile if enabled.
+Nil means fd is not installed or should not be used."
+  :type 'string)
```

Comment:
The new defcustoms should also have a `:package-version` property.

Reply:
https://github.com/bbatsov/projectile/pull/1825/commits/e49fee486cbb1e3d8caf446c91d057a7ffcb0f93